validphys2/src/validphys/app.py

```diff
@@ -147,11 +144,11 @@ def upload_context(do_upload, output):
         return contextlib.ExitStack()
 
     def run(self):
-        if sys.version_info < (3, 6):
+        if sys.version_info < (3, 9):
```

Comment:
Did we officially drop 3.8 as well?

Reply:
Yes. We are no longer building conda packages for 3.8.

Reply:
Ok, I wonder if we should drop also from the other projects...

Point is that I'm fully happy in dropping 3.7, that will go EOL in June (now that I'd like to support 3.11, I don't feel guilty dropping an *almost* EOL version, even if it's the one running on `galileo`), but I'm not as sure for 3.8.

In particular, I believe @cschwan for example is still using it. Most likely he doesn't care too much about installing vp, but it makes me think that some still very relevant versions of major distributions are still on it.

Reply:
Yes, some are. I still have 3.8 in a few places. And if you really want to use it you can just install n3fit/validphys manually and it will work. But since we are not supporting it anymore in "the official conda package" we should warn people.